tsconfig.json

```diff
@@ -1,11 +1,8 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
-    "outDir": "./lib/cjs",
-    "declaration": true,
-    "noEmitOnError": true,
-    "noImplicitAny": true
-  },
-  "include": ["src/**/*"]
+    "compilerOptions": {
+        "lib": ["es2015", "dom"],
```

Comment:
I spent half of this morning trying to figure just thing very thing 🤦‍♂️ 

Am I correct that that this does *not* include any polyfills but just adds support in the types?

Reply:
I'm no TS guru but from what I've gathered it adds those to the ts compile process when its looking up which possible types it you can use in your runtime. 

https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#including-built-in-type-declarations-with---lib

Reply:
Correct, this does not include polyfills.

Reply:
Thanks @DanielRosenwasser!